contracts/Registry.sol

```diff
@@ -7,7 +7,7 @@ contract Registry is IRegistry, ERC165 {
 
     struct OrganizationRegistration {
         bytes32 organizationId;
-        string organizationName;
+        bytes orgMetadataURI;
```

Comment:
I understand would be ok to have dynamic bytes here, but I would go with a fixed-length.

An important practical difference is that the fixed length bytes32 can be used in function arguments to pass data in or return data out of the contract. The variable length bytes type can be a function argument also, but only for internal use (inside the same contract) because the interface, called the ABI, doesn't support variable length types.

Reply:
@tiero Completely agree from best practices point of view. As we are planning to store the IPFS Hash on the same lines as Service/TypeRepo Metadata Hash where the length might go beyond the bytes32 size based on the IPFS documentation.

Reply:
ah I see. Ok so let's keep it dynamic